package.json

```diff
@@ -43,33 +43,33 @@
   "readmeFilename": "README.md",
   "peerDependencies": {
     "events": "^3.1.0",
-    "react": "^16.13.0 || ^17.0.0",
-    "react-native": "^0.63.0"
+    "react": "^17.0.2",
+    "react-native": "^0.66.0"
```

Comment:
try something like `>= 0.63`, we still want to keep the old version of react-native work.

Reply:
You may also want to revert the change for `react` dep, after the change, the package won't compatible with react v16.13 and above.